Mathlib/Data/Polynomial/EraseLead.lean

```diff
@@ -123,6 +124,21 @@ theorem eraseLead_card_support' {c : ℕ} (fc : f.support.card = c + 1) :
   eraseLead_card_support fc
 #align polynomial.erase_lead_card_support' Polynomial.eraseLead_card_support'
 
+theorem eraseLead_card_support_one (h : f ≠ 0) : f.eraseLead.support.card + 1 = f.support.card := by
```

Comment:
```suggestion
theorem card_support_eraseLead_add_one (h : f ≠ 0) : f.eraseLead.support.card + 1 = f.support.card := by
```
and can you also provide
```
@[simp]
theorem card_support_eraseLead : f.eraseLead.support.card = f.support.card - 1 := by
```

Reply:
The other one you suggest is pretty much the same as the existing
```
theorem eraseLead_card_support {c : ℕ} (fc : f.support.card = c) :
    f.eraseLead.support.card = c - 1 := by
```
and the proof would just be
```
@[simp]
theorem card_support_eraseLead : f.eraseLead.support.card = f.support.card - 1 :=
  eraseLead_card_support rfl
```
Does that still merit adding?

Reply:
Can you just change `eraseLead_card_support`? This is a very weird statement.